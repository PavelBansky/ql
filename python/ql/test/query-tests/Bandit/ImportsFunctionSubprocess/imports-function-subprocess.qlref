Bandit/imports-function-subprocess.ql