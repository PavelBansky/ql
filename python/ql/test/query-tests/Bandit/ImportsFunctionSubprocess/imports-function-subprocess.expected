| Test.py:6:14:6:37 | __import__() | Consider possible security implications associated with subprocess module |
| Test.py:19:5:19:38 | Attribute() | Consider possible security implications associated with subprocess module |
